CONTRIBUTING.md

```diff
@@ -44,6 +44,14 @@ In this case the reviewer or PR maintainer needs to make sure to create an issue
 
 Find here [a guide](doc/setup/SETUP.md) on how to setup your local development environment.
 
+## Route Naming Conventions
+
+- Always use **kebab-case** (e.g. 'exampleAssessment' → 'example-assessment')
+- Before id there should always be the name of the entity to which this id relates to
+- Use **plural for server route's** entities and **singular for client route's** entities
+- Key entity at the end of the route (e.g. text-editor/participations/{participationId} → participations/{participationId}/text-editor)
```

Comment:
We need to make sure that the client routes are a good fit for the module structure, enabling us to switch over to lazy loading easily. So this should be compatible with #736.